quizzes/quiz_1.md

```diff
@@ -0,0 +1,59 @@
+
+{quiz, id: introduction, attempts: 10}
+
+## Introduction Quiz
+
+Choose the best answer for the following questions.
+
+{choose-answers: 4}
+? What is one of the benefits of data sharing mentioned in the Introduction?
+
+C) Increased collaboration opportunities
+C) Improved reproducibility and rigor
+C) Generally greater impact and benefit for society
+C) A greater number of citations to articles
+m) Decreased transparency in research
+m) Limited reproducibility of study findings
+o) Data storage costs covered by the federal government
+o) Scientific analysis is limited to experts in the field
+
+{choose-answers: 4}
+? Is a DMS plan is required?
+
+m) Yes: All grant mechanisms require a DMS plan.
+C) Maybe: Look up the NIH Activity Code to see if you are required to submit a DMS Plan.
+m) No: The DMS plan is optional, but can improve your grant score.
+m) Yes: Ask the Office of Sponsored Research for your institution to prepare a plan.
+
+{choose-answers: 4}
+? Which of the following is NOT considered scientific data under the NIH Data Management and Sharing (DMS) Policy?
+
+m) Unpublished results
+m) Null results
+C) Lab notebooks
+C) Plans for future research
+C) Peer reviews
+m) Results used to publish papers
+
+{choose-answers: 4}
+? Which of the following is FALSE regarding the grant review process and your DMS Plan??
+
+m) Reviewers will not have access to your DMS plan.
+m) Reviewers will see your budget which will include some descriptions of how money will be spent to manage and share data.
+m) DMS plan should not influence your grant score.
+C) The Program Officer will write the DMS plan and will work with the PI to address any concerns.
+C) The Program Officer will score the plan and add the value to your grant score.
```

Comment:
C) The Program Officer will score the plan and add the value to your grant score.

I think this one is an incorrect answer